src/bundle/Resources/config/routing_rest.yaml

```diff
@@ -14,7 +14,7 @@ ezplatform.bulk_operation:
 # Location Tree
 #
 
-ezplatform.location.tree.load_children:
+ibexa.rest.location.tree.load_children:
     # @todo change name to content tree
```

Comment:
Could be also be the time to either do or remove those todos?

Reply:
This is out of scope. This is just route name rebranding.